larray/inout/excel.py

```diff
@@ -249,13 +256,26 @@ def _dump_item(self, key, value, *args, **kwargs):
         else:
             raise TypeError()
 
+    def _read_metadata(self):
+        sheet_meta = '__metadata__'
+        if sheet_meta in self.handle.sheet_names:
+            s = pd.read_excel(self.handle, sheet_meta, index_col=[0], squeeze=True)
+            return _series_to_metadata(s)
+        else:
+            return Metadata()
+
+    def _dump_metadata(self, metadata):
+        if len(metadata) > 0:
+            s = _metadata_to_series(metadata)
+            s.to_excel(self.handle, '__metadata__', engine='xlsxwriter')
+
     def save(self):
         if len(self.axes) > 0:
             df = _axes_to_df(self.axes.values())
-            df.to_excel(self.handle, '__axes__', engine='xlsxwriter')
+            df.to_excel(self.handle, '__axes__', index=False, engine='xlsxwriter')
```

Comment:
is this change really related to session metadata???

Reply:
Nope, sorry.